hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -60,14 +61,39 @@ public HoodieCommitArchiveLog(HoodieWriteConfig config, FileSystem fs) {
      */
     public boolean archiveIfRequired() {
         List<HoodieInstant> commitsToArchive = getCommitsToArchive().collect(Collectors.toList());
```

Comment:
lets rename getCommitsToArchive also consistently? instants can be commits, compactions, delta commits from what I can see..